File: EstruturaSequencial/Ex11/Ex11.c
#include <iostream.h>
#include <conio.h>
#include <math.h>
void main()
{float num, quad, cubo, r2, r3; 
clrscr();
cout << "\n Digite um numero:";
cin >> num; 
quad = pow(num, 2);
cubo = pow(num. 3);
r2 = sqrt(num);
r3 = exp(log(num) * 1/3);
cout << "\n O numero ao quadrado é:" << quad;
cout << "\n O numero ao cubo é:" << cubo;
cout << "\n A raiz quadrada do numero é: " << r2;
cout << "\n A raiz cubica do numero é: " << r3;
getch();
} 

File: EstruturaSequencial/Ex11/Ex11.pas
PROGRAM EX3;
USES CRT;
VAR  num, quad, cubo, r2, r3; 
BEGIN
CLRSCR;
WRITELN ('Digite um número:');
READLN(num);
quad := SQR(num);
cubo := EXP(3 * LN(num)); 
r2 := SQRT(num); 
r3 := EXP(1/3 * LN(num)); 
WRITELN('Número ao quadrado  = ',quad:5:2);
WRITELN('Número ao cubo  = ',cubo:5:2);
WRITELN('Raiz quadrada do número  = ',r2:5:2);
WRITELN('Raiz cubica do número  = ',r3:5:2);